src/main/java/org/dave/compactmachines3/gui/framework/widgets/WidgetInputField.java

```diff
@@ -50,7 +50,7 @@ public WidgetInputField(String fieldIdentifier) {
         this.addListener(MouseClickEvent.class, new IWidgetListener<MouseClickEvent>() {
             @Override
             public WidgetEventResult call(MouseClickEvent event, Widget widget) {
-                //Logz.info("Mouse clicked @ x=%d, y=%d", event.x, event.y);
+                //Logz.info("Mouse clicked @ x={}, y={}", event.x, event.y);
```

Comment:
Prefer full removal of unnecessary logging

Reply:
Noted